google_auth_oauthlib/flow.py

```diff
@@ -107,6 +112,7 @@ def __init__(self, oauth2session, client_type, client_config):
         """Mapping[str, Any]: The OAuth 2.0 client configuration."""
         self.oauth2session = oauth2session
         """requests_oauthlib.OAuth2Session: The OAuth 2.0 session."""
+        self.redirect_uri = redirect_uri
```

Comment:
Do you want a docstring for this?

Reply:
There's already one for it below (it's a property)